Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_CPPException.cpp

```diff
@@ -94,6 +94,17 @@ __cxa_throw(void *thrown_exception, std::type_info *tinfo, void (*dest)(void *))
 }
 }
 
+void
+sentrycrashcm_cppexception_callOriginalTerminationHandler(void)
+{
+    // Can be NULL as the return value of set_terminate can be a NULL pointer; see:
+    // https://en.cppreference.com/w/cpp/error/set_terminate
+    if (g_originalTerminateHandler) {
```

Comment:
I would prefer if this were written more explicitly:
```suggestion
    if (g_originalTerminateHandler != NULL) {
```